webapp/templates/jury/analysis/contest_overview.html.twig

```diff
@@ -397,14 +397,30 @@ submissions.forEach(submission => {
   }
 });
 
-for (let minute = 0; minute < contest_duration_minutes; minute++) {
+for (let minute = 0; minute <= contest_duration_minutes; minute++) {
   let this_minute_submission_nums = 0;
   submission_stats.forEach(stat => {
     this_minute_submission_nums += stat.values[minute][1];
   });
   max_submissions_per_minute = Math.max(max_submissions_per_minute, this_minute_submission_nums);
 }
 
+function generate_values(start_point, end_point, number_of_points) {
```

Comment:
I think we can this both simpler and nicer:

This will make sure that the ticks we use are still limited but are nice. In the default 5h long contest, every 15 minutes there is a tick, for longer contests, it increases to 30mins, 60mins and so on.

```
// Pick a nice round tickDelta and tickValues
var tickDelta = 15;
while (contest_duration_minutes / tickDelta > 15) {
  tickDelta *= 2;
}
var tickValues = Array.from({ length: Math.ceil(contest_duration_minutes / tickDelta) + 1 }, (_, i) => i * tickDelta);
```

Reply:
So, the unit will always be in minutes?

If we take a classroom scenario, it's quite common for a contest to last an entire semester.

Reply:
You are right that we also would want to fall back to larger units than minutes when the contest duration is that long - I assume we should do that in the same change that adds the limitation to 300 buckets/bars. Do you rather want to do this in this PR or separately?

Reply:
Let's handle it separately.

Reply:
alredy update